packages/eas-cli/src/commands/channel/create.ts

```diff
@@ -127,10 +83,14 @@ export default class ChannelCreate extends Command {
       branchMessage = `We also went ahead and made a branch with the same name`;
     }
 
-    const newChannel = await createUpdateChannelOnAppAsync({
+    const newChannel = await ChannelMutation.createForAppAsync({
       appId: projectId,
-      channelName,
-      branchId,
+      name: channelName,
+      // Point the new channel at a branch with its same name.
+      branchMapping: JSON.stringify({
```

Comment:
We should probably make a utility for this, to translate a set of (+ possible rollout options?) to a `branchMapping` JSON.